### DISCUSSION TITLE
streamlabs "OBS" fiasco?

### DISCUSSION THREAD

**Discussion by kdpuvvadi** [microsoft/winget-pkgs#36207] [General]
with all the controversy &  as streamlabs is dropping obs from it's software, should the identifier also change here, drop the obs from moniker also?

**Comment by vedantmgoyal9** [comment]
I don't think we should change `PackageName` field?

**Reply by kdpuvvadi** [reply]
streamlab is dropping obs from their branding and their software. 
Here's the statement

https://twitter.com/streamlabs/status/1461044715634135046
